bot.py

```diff
@@ -663,6 +687,16 @@ def main_color(self) -> int:
                         self.blocked_users.pop(str(message.author.id))
                 else:
                     logger.debug("User blocked, user %s.", message.author.name)
+        elif cooldown > now:
+            delta = human_timedelta(cooldown)
```

Comment:
Don't believe `reaction` is defined.

Reply:
And also, why don't you implement it the same way as account_age and guild_age, although they look a bit messy. The reason for how the others are implemented is to log it in the block reason and only show the "Message not sent" message once in a user's DM. If you want, just leave it and I can convert it later on.